css/properties/place-content.json

```diff
@@ -47,6 +47,54 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "box-alignment-updates": {
```

Comment:
The id is a bit cryptic: what if there are further updates.

I propose "prevent-duplicated-values" (but there may be better values)

Reply:
agreed; changed